### ISSUE TITLE
[Warrior] RFDT formula bugged

### ISSUE THREAD

**Issue by siralia** [simulationcraft/simc#3138]
I made a simulation with an equal amount of dmg taken (also distribution wise) in simc compared to one of my mythic xavius kills. I'm getting significantly more rage from RFDT in simc compared logs.
I tried the RFDT formula on the sim and it seems it factors in too much dmg, compared the actual dmg taken. Doing the math i got a RFDT result very close to what sim showed me if i factored in the dmg lost trough blocks.

TL;DR i think blocked dmg counts towards RFDT on sims
@Collisionc

**Comment by Collisionc** [comment]
Thanks, that was kind of a big issue... lots of rage.
